i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx

```diff
@@ -0,0 +1,137 @@
+
+```typescript
+it("test2", async () => {
+    new Promise((resolve, reject) => {
+        setTimeout(() => {
+            reject("Something goes wrong");
+        }, 30000);
+    });
+});
+```
+
+This test will immediately pass, and only after 30 seconds (assuming there are still tests in the queue to run) will an `Unhandled rejection` error appear.
+
+Therefore, to find the problematic test, you will need to look at the execution log and find the last completed tests before the error information. The error will look something like this:
+
+```bash
+[13:48:57 GMT+3] Unhandled Rejection in hermione:worker:10830:
+Promise: {}
+Reason: Something goes wrong
+[13:48:57 GMT+3] Terminating on critical error: Unhandled Rejection in hermione:worker:10830:
+Promise: {}
+Reason: Something goes wrong
+```
+
+#### test.id and suite.id Are Now Properties {#test_id_and_suite_id_are_properties}
+
+The `id` for a test and suite is now a property, not a method, and should be used as `test.id` or `suite.id`. The `test.id` call is usually used in the hermione config to define the [screenshotsDir][screenshots-dir] option.
+
+#### Removed the saveHistory Option from the Config {#removed_save_history_option_from_config}
+
+The [saveHistory][save_history] option has been removed from the config. You now need to use [saveHistoryMode][save_history_mode] with available values: `all`, `none`, `onlyFailed`. The default is `all`, meaning history is saved for all tests. So, you can skip explicitly setting this option.
+
+### Minor Changes {#minor_changes}
+
+-   Upgraded from [webdriverio@7][webdriverio@7] to [webdriverio@8][webdriverio@8];
+-   Increased test reading speed by about 3 times: tests are now read once in the master (previously read separately for each browser);
+-   Added the ability to write the config in TS. You can create `.hermione.conf.ts`, and hermione will compile and read it (assuming your project includes `ts-node`);
+-   Added timestamps to hermione logs, looking like:
+
+    ```bash
+    [13:48:09 GMT+3] ✓ suite test2 [chrome-desktop:SESSION_ID] - 875ms
+    ```
+
+### Patch Changes {#patch_changes}
+
+-   Properly supported taking screenshots on devices with fractional `pixelRatio` (e.g., `Google Pixel`). Previously, screenshots on such devices were incorrect;
+-   Moving the cursor away using the `resetCursor: true` option no longer triggers an error if the top-left corner of the `body` element has negative coordinates.
+
+## How to Migrate? {#how_to_move}
+
+### 1. Update hermione to 7+ and Plugins to the Latest Versions {#update_hermione_and_plugins}
+
+Specifically:
+
+-   [hermione][hermione]
```

Comment:
It would be nice to replace all of expired "hermione" references as well, but it is hard, because some of them should be "hermione". Old plugins, for example.

Reply:
I did replace all mentions of hermione with testplane. Hermione is here because it's a migration guide for the old version.